docker-entrypoint.sh

```diff
@@ -41,18 +41,18 @@ function start_service {
     check_running_status $1
 }
 
-jwt_secret_file="/etc/slurm/jwt/jwt_hs256.key"
-if [ "$JWT_SECRET" ]
+jwt_key_file="/etc/slurm/jwt/jwt_hs256.key"
+if [ "$SLURM_JWT_KEY" ]
 then
     echo "- JWT secret variable found, writing..."
-    echo "$JWT_SECRET" > $jwt_secret_file
+    echo "$SLURM_JWT_KEY" > $jwt_key_file
```

Comment:
echo will automatically add a newline - not sure whether this is the problem you were seeing, but you can do `echo -n "$SLURM_JWT_KEY"` to prevent echo from adding the newline to the end.

Reply:
Ace - done